src/w3c/headers.js

```diff
@@ -501,17 +501,16 @@ export function run(conf, doc, cb) {
       $("<meta lang='' property='dc:language' content='en'>")
     );
   }
-  // insert into document and mark with microformat
-  var bp;
-  if (conf.isCGBG) bp = cgbgHeadersTmpl(conf);
-  else bp = headersTmpl(conf);
-  $("body", doc)
-    .prepend($(bp))
-    .addClass("h-entry");
+  // insert into document
+  document.body.insertBefore(
+    (conf.isCGBG ? cgbgHeadersTmpl : headersTmpl)(conf),
```

Comment:
A lot is happening here, might be a a little easier on the eyes/easier to debug if things go bad if:

```JS
const tmpl = conf.isCGBG ? cgbgHeadersTmpl : headersTmpl;
const elem = tmpl(conf);
document.body.insertBefore(elem, document.body.firstChild) 
```

`elem` should probably be named whatever comes out of template? you said it was a div, right?